build.gradle

```diff
@@ -72,42 +73,67 @@ dependencies {
 	testCompile 'org.mockito:mockito-all:2.0.2-beta'
 }
 
-task allPlatform(type: Zip, group: 'release', dependsOn: ':shadowJar') {
+def assetsDirectory = file("${buildDir}/assets")
+git {
```

Comment:
I think this could pose a problem, cloning all of the assets repo. The assets repo will continue to grow with items that are not necessarily needed by install4j. For example, wav files are likely to be parked there which will add 150-200MB of content.  My concern is that this would slow down the build in order to download a few hundred megabytes of unnecessary data.

If we want to keep things simple for the download, adding a build to the asset repo so it publishes zip files IMO would be the best way to go. Alternatively just list out explicitly each file that needs to be downloaded. Managing the list of a dozen files for now I think is okay.